src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,70 @@
-export const Tabs = () => {};
+import React, { useState } from 'react';
+
+export interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const [toggleState, setToggle] = useState(tabs[0]);
+
+  const chooseTab = (tab: Tab) => {
```

Comment:
The wrapper seems useless - you may use `setToggle` (renamed) in `tabClick` function